client/src/pages/SignUp/SignUpForm/SignUpForm.tsx

```diff
@@ -31,6 +34,16 @@ interface Props {
 
 const SignUpForm = ({ handleSubmit }: Props): JSX.Element => {
   const classes = useStyles();
+  const { updateLoginContext } = useAuth();
+  let demo = false;
+  const demoLogin = () => {
+    login(demoValues.email, demoValues.password).then((data) => {
+      if (data.success) {
```

Comment:
If there is an error maybe we can update the snack bar (useSnackbarContext) to display the error message ? :)

Reply:
Will add, thanks!